perf.groovy

```diff
@@ -27,16 +27,11 @@ def static getOSGroup(def os) {
 // Setup perflab tests runs
 [true, false].each { isPR ->
     ['Windows_NT'].each { os ->
-        ['x64', 'x86', 'x86jit32'].each { arch ->
+        ['x64', 'x86' ].each { arch ->
             def architecture = arch
             def testEnv = ''
 
-            if (arch == 'x86jit32')
-            {
-                architecture = 'x86'
-                testEnv = '-testEnv %WORKSPACE%\\tests\\x86\\compatjit_x86_testenv.cmd'
-            }
-            else if (arch == 'x86')
+            if (arch == 'x86')
```

Comment:
Do we need to keep this? This was put in on the off chance that we switched back to jit32 being the main jit.